docs/source/user/saml.rst

```diff
@@ -36,7 +36,7 @@ a SSO.
 Setup
 -----
 
-Install required system dependencies:
+Install required system dependencies::
```

Comment:
```suggestion
Install required system dependencies:
```

Reply:
the two semi-colons are needed to format the next line as a code block